src/workerd/api/crypto-impl-asymmetric.c++

```diff
@@ -1863,8 +1867,9 @@ kj::Own<CryptoKey::Impl> CryptoKey::Impl::importEcdsa(
       return importAsymmetric(
           js, format, kj::mv(keyData), normalizedName, extractable, keyUsages,
           // Verbose lambda capture needed because: https://bugs.llvm.org/show_bug.cgi?id=35984
-          [curveId = curveId](SubtleCrypto::JsonWebKey keyDataJwk) -> kj::Own<EVP_PKEY> {
-        return ellipticJwkReader(curveId, kj::mv(keyDataJwk));
+          [curveId = curveId, normalizedName=kj::str(normalizedName)]
```

Comment:
Nit: Use spaces around the `=`. I don't see an upstream PR – while it's debatable whether it's needed for a small change like this, it might have flagged the formatting here. Same applies to the other two instances below.

`kj::str(normalizedName)` is used to avoid issues in case normalizedName goes out of scope in the calling function, right?
PR LGTM otherwise.

Reply:
> kj::str(normalizedName) is used to avoid issues in case normalizedName goes out of scope in the calling function, right?

Yep, largely precautionary.